scene/gui/graph_edit.cpp

```diff
@@ -270,78 +273,78 @@ void GraphEdit::_update_scroll_offset() {
 	set_block_minimum_size_adjust(true);
 
 	for (int i = 0; i < get_child_count(); i++) {
-		GraphNode *gn = Object::cast_to<GraphNode>(get_child(i));
-		if (!gn) {
+		GraphControl *graph_node = Object::cast_to<GraphControl>(get_child(i));
+		if (!graph_node) {
 			continue;
 		}
 
-		Point2 pos = gn->get_position_offset() * zoom;
-		pos -= Point2(h_scroll->get_value(), v_scroll->get_value());
-		gn->set_position(pos);
-		if (gn->get_scale() != Vector2(zoom, zoom)) {
-			gn->set_scale(Vector2(zoom, zoom));
+		Point2 pos = graph_node->get_position_offset() * zoom;
+		pos -= Point2(h_scrollbar->get_value(), v_scrollbar->get_value());
+		graph_node->set_position(pos);
+		if (graph_node->get_scale() != Vector2(zoom, zoom)) {
+			graph_node->set_scale(Vector2(zoom, zoom));
 		}
 	}
 
-	connections_layer->set_position(-Point2(h_scroll->get_value(), v_scroll->get_value()));
+	connections_layer->set_position(-Point2(h_scrollbar->get_value(), v_scrollbar->get_value()));
 	set_block_minimum_size_adjust(false);
 	awaiting_scroll_offset_update = false;
 
-	if (!setting_scroll_ofs) { //in godot, signals on change value are avoided as a convention
-		emit_signal(SNAME("scroll_offset_changed"), get_scroll_ofs());
+	// In godot, signals on change value are avoided as a convention.
+	if (!setting_scroll_ofs) {
+		emit_signal(SNAME("scroll_offset_changed"), get_scroll_offset());
 	}
 }
 
 void GraphEdit::_update_scroll() {
 	if (updating) {
 		return;
 	}
-
 	updating = true;
 
 	set_block_minimum_size_adjust(true);
 
-	Rect2 screen;
+	Rect2 screen_rect;
 	for (int i = 0; i < get_child_count(); i++) {
-		GraphNode *gn = Object::cast_to<GraphNode>(get_child(i));
-		if (!gn) {
+		GraphControl *graph_node = Object::cast_to<GraphControl>(get_child(i));
```

Comment:
Same issue with the variable name. Probably a bunch of others around all the touched files.